### PR TITLE
enable seing a multisig account's transaction list, fixes #84

### LINKED ISSUE #84: enable seing a multisig account's transaction list

### PR DESCRIPTION
**Pull request by decentraliser**

### CODE REVIEW

File: src/components/TransactionList/TransactionTable/TransactionTable.vue [hunk 1]
```diff
@@ -11,7 +15,7 @@
         />
       </div>
     </div>
-    <div v-if="!transactions.length" class="no-data-outer-container">
+    <div v-if="!transactions.length && !fetchingTransactons" class="no-data-outer-container">
```
Comment (evias) [inline src/components/TransactionList/TransactionTable/TransactionTable.vue]: typo `fetchingTransactions`

File: src/components/TransactionList/TransactionTable/TransactionTable.vue [hunk 2]
```diff
@@ -40,12 +45,20 @@ import TransactionListHeader from '@/components/TransactionList/TransactionListH
     TransactionRow,
     TransactionListHeader,
   },
+  computed: mapGetters({ fetchingTransactons: 'app/fetchingTransactons' }),
```
Comment (evias) [inline src/components/TransactionList/TransactionTable/TransactionTable.vue]: typo `fetchingTransactions`

File: src/components/TransactionList/TransactionTable/TransactionTable.vue [hunk 3]
```diff
@@ -40,12 +45,20 @@ import TransactionListHeader from '@/components/TransactionList/TransactionListH
...
   @Prop({ default: [] }) transactions: Transaction[]
   @Prop({ default: 'no_data_transactions'}) emptyMessage: string
   public nodata = [...Array(10).keys()]
 
+  /**
+   * Whether transactios are currently being fetched
+   * @protected
+   * @type {boolean}
+   */
+  protected fetchingTransactons: boolean
```
Comment (evias) [inline src/components/TransactionList/TransactionTable/TransactionTable.vue]: typo `fetchingTransactions`

File: src/components/TransactionListOptions/TransactionListOptionsTs.ts [hunk 4]
```diff
@@ -110,34 +133,7 @@ export class TransactionListOptionsTs extends Vue {
...
-      const service = new WalletService(this.$store)
-      return self.concat(...multisigInfo.multisigAccounts.map(
-        ({publicKey}) => ({
-          publicKey,
-          label: service.getWalletLabel(publicKey, this.networkType) + this.$t('label_postfix_multisig'),
-        })))
-    }
-
-    return self
+    return new MultisigService(this.$store).getSigners(`${this.$t('label_postfix_multisig')}`)
```
Comment (evias) [inline src/components/TransactionListOptions/TransactionListOptionsTs.ts]: remove text paremeter, getSigners can use the translation module

File: src/store/AppInfo.ts [hunk 5]
```diff
@@ -41,6 +41,7 @@ export default {
     faucetUrl: networkConfig.faucetUrl,
     defaultFee: feesConfig.normal,
     defaultWallet: '',
+    fetchingTransactons: false,
```
Comment (evias) [inline src/store/AppInfo.ts]: typo `fetchingTransactions`

File: src/store/AppInfo.ts [hunk 6]
```diff
@@ -56,6 +57,7 @@ export default {
     faucetUrl: (state) => state.faucetUrl,
     defaultFee: (state) => state.defaultFee,
     defaultWallet: (state) => state.defaultWallet,
+    fetchingTransactons: (state) => state.fetchingTransactons,
```
Comment (evias) [inline src/store/AppInfo.ts]: typo `fetchingTransactions`

File: src/store/AppInfo.ts [hunk 7]
```diff
@@ -74,6 +76,7 @@ export default {
     },
     setDefaultFee: (state, maxFee) => Vue.set(state, 'defaultFee', maxFee),
     setDefaultWallet: (state, defaultWallet) => Vue.set(state, 'defaultWallet', defaultWallet),
+    setFetchingTransactions: (state, bool: boolean) => Vue.set(state, 'fetchingTransactons', bool),
```
Comment (evias) [inline src/store/AppInfo.ts]: typo `fetchingTransactions`

File: src/components/TransactionListOptions/TransactionListOptionsTs.ts (hunk 4 above)
Comment (decentraliser) [reply src/components/TransactionListOptions/TransactionListOptionsTs.ts]: passed i18n to an optional argument of multisigService constructor

File: src/store/AppInfo.ts [hunk 8]
```diff
@@ -56,6 +57,7 @@ export default {
     faucetUrl: (state) => state.faucetUrl,
     defaultFee: (state) => state.defaultFee,
     defaultWallet: (state) => state.defaultWallet,
+    fetchingTransactions: (state) => state.fetchingTransactions,
```
Comment (evias) [inline src/store/AppInfo.ts]: `isFetchingTransactions` seems to be more natural as indicating "boolean" type with `is` annotation

File: src/services/MultisigService.ts [hunk 9]
```diff
@@ -32,13 +34,19 @@ export class MultisigService extends AbstractService {
...
+  public $i18n: VueI18n
   /**
    * Construct a service instance around \a store
    * @param store
    */
-  constructor(store?: Store<any>) {
+  constructor(store?: Store<any>, i18n?:  VueI18n) {
     super()
     this.$store = store
+    this.$i18n = i18n
```
Comment (evias) [inline src/services/MultisigService.ts]: move into `AbstractService`

**Pull request merged**